### DISCUSSION TITLE
LocalTuya

### DISCUSSION THREAD

**Discussion by goskim** [make-all/tuya-local#1771] [Q&A]
Hi, i have some problems with configuration of double curtain switch Loratap , i have tried a lot options (in Localtuya) , but it doesnt work, i dont understand the numbers, and which one i should choose , i have also curtain motor and i want to connect the both of this devices
![IMG_20240324_101333](https://github.com/make-all/tuya-local/assets/97799140/a9f99b85-b51a-4aef-860c-c0957f4cdcb3)

**Comment by make-all** [comment]
This is the wrong venue to ask questions about localtuya
